pdns/dnsdist-carbon.cc

```diff
@@ -104,12 +104,20 @@ try
           str<<base<<"tcpavgqueriesperconnection" << ' '<< state->tcpAvgQueriesPerConnection.load() << " " << now << "\r\n";
           str<<base<<"tcpavgconnectionduration" << ' '<< state->tcpAvgConnectionDuration.load() << " " << now << "\r\n";
         }
+
+        std::map<std::string,uint64_t> frontendDuplicates;
```

Comment:
Perhaps `typedef` this?

Reply:
I'm not sure that would make the code more readable, since it's a quite simple type, and you would then have to lookup the typedef definition if you want to known what the actual type is.